alma/views/templates/hook/displayPaymentReturn.tpl

```diff
@@ -46,7 +46,11 @@
                 {elseif $smarty.foreach.counter.iteration === 2}
                     {l s='At shipping' mod='alma'}
                 {else}
-                    {l s='%s month later' sprintf=[$plan->time_delta_from_start['months']] mod='alma'}
+                    {assign var='nb_installment_month' value=$plan->time_delta_from_start['months']}
```

Comment:
a good candidate to make your first test E2E 😉

Reply:
Do you want me to implement it in this branch or is it just for info for a future branch?

Reply:
E2E are stored in an other repository ... we will speak about that during our meeting this afternoon